VolleyPaint/Assets/Blobcreate/Projectile Toolkit/Core/TrajectoryPredictor.cs

```diff
@@ -14,11 +14,10 @@ public class TrajectoryPredictor : MonoBehaviour
 		public LineRenderer Line { get => line; set => line = value; }
 		public Vector3[] Positions => positions;
 		public int PositionCount => validCount;
-
-		/// <summary>
-		/// The gravity that used to simulate trajectories. Currently does not support custom gravity direction.
-		/// </summary>
-		public Vector3 Gravity
+        /// <summary>
+        /// The gravity that used to simulate trajectories. Currently does not support custom gravity direction.
+        /// </summary>
+        public Vector3 Gravity
```

Comment:
the spacing seems messed up here, you might want to change it back

Reply:
Not sure what happened tbh